app/server/utils/phone.py

```diff
@@ -83,4 +85,18 @@ def send_messagebird_message(self, to_phone, message):
 
     def send_at_message(self, to_phone, message):
         if to_phone:
-            self.africastalking_client.send(message, [to_phone])
+
+            # First try with custom sender Id
+            resp = self.africastalking_client.send(
+                message,
+                [to_phone],
+                sender_id=current_app.config.get('AT_SENDER_ID', None)
+            )
+
+            # If that fails, fallback to no sender ID
+            if resp['SMSMessageData']['Message'] == 'InvalidSenderId':
+                server.sentry.captureMessage("InvalidSenderId {}".format(current_app.config.get('AT_SENDER_ID', None)))
```

Comment:
nice, I need to use sentry more :)

Reply:
yeah i think we need to make it a deeper part of our processs - like making sure we tick off bugs that get raised etc.